### ISSUE TITLE
RubyReaper treats runtime lock as orphan and delete it

### ISSUE THREAD

**Issue by yaoyi** [mhenrixon/sidekiq-unique-jobs#580]
when using until_and_while_executing, RubyReaper treats runtime lock as orphan and delete it, which causes duplicate running jobs.

in the follow codes, digest in workers is without suffix "RUN"
```ruby
def active?(digest) # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
        Sidekiq.redis do |conn|
          procs = conn.sscan_each("processes").to_a
          return false if procs.empty?

          procs.sort.each do |key|
            valid, workers = conn.pipelined do
              conn.exists(key)
              conn.hgetall("#{key}:workers")
            end

            next unless valid
            next unless workers.any?

            workers.each_pair do |_tid, job|
              next unless (item = safe_load_json(job))

              payload = safe_load_json(item[PAYLOAD])

              return true if payload[LOCK_DIGEST] == digest
              return true if considered_active?(payload[CREATED_AT])
            end
          end

          false
        end
      end
```

**Comment by mhenrixon** [comment]
Thanks for the report @yaoyi, I'll see what I can do about that!

**Comment by mhenrixon** [comment]
Released as v7.0.5
